submissions/fabunmiTolulope/php/Number_loops.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+// This program will display every number between 100 and 150 using while loop
+$x = 100;
+while ($x <= 150) {
```

Comment:
The question says between `100 and 150`

Reply:
I will effect the changes as soon as I can... @adaugochi

Reply:
Between includes 100 and 150

Reply:
@ToluAkin I have changed it from <= to <. Thanks